examples/browser/test/right-panel/right-panel.ts

```diff
@@ -0,0 +1,57 @@
+/********************************************************************************
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import "webdriverio";
+
+export class RightPanel {
+
+    constructor(protected readonly driver: WebdriverIO.Client<void>) { }
+
+    doesTabExist(tabName: string): boolean {
+        return this.driver.element(`.p-TabBar.theia-app-right .p-TabBar-content`).isExisting(`div\=${tabName}`);
```

Comment:
```ts
`div\=thing` === `div=thing`
```
I was looking for a special meaning for that backslash, but couldn't find anything... It might be a detail but it is a bit confusing at first. Should we just write `'div=${tabname}`?

Reply:
These instances were copied from front-panel. I added a commit to remove all those backslashes.